src/components/pages/loanOffer/personalInfo/index.tsx

```diff
@@ -1,143 +1,224 @@
+                                                    )}
+                                                />
+                                            </Margin>
+                                            <Margin bottom={32}>
+                                                <Field
+                                                    name="email"
+                                                    validate={composeValidators(
+                                                        requiredField,
+                                                        emailFormat
+                                                    )}
+                                                    render={({
+                                                        input,
+                                                        meta,
+                                                    }) => (
+                                                        <React.Fragment>
+                                                            <TextField
+                                                                type="email"
+                                                                label="Email"
+                                                                placeholder="Enter your email"
+                                                                {...input}
+                                                                {...meta}
+                                                            />
+                                                            {meta.touched &&
+                                                                meta.error && (
+                                                                    <FieldError>
+                                                                        {
+                                                                            meta.error
+                                                                        }
+                                                                    </FieldError>
+                                                                )}
+                                                        </React.Fragment>
+                                                    )}
+                                                />
+                                            </Margin>
+                                            {/* <Margin vertical={32}>
+                                                <Checkbox
+                                                    id="videoInterview"
+                                                    name="videoInterview"
+                                                    label="I would like to have a video interview with the borrower"
+                                                />
+                                            </Margin> */}
+                                            <Row>
+                                                {/* <Col lg={6} md={12}>
```

Comment:
should this be deleted, since its one step, now?

Reply:
Yup! WIP